docs/processes/cldr-icu.md

```diff
@@ -380,11 +380,11 @@ If the data is OK , other sources of failure can include:
 Exhautive tests should always be run for a CLDR-ICU integration PR before it is merged.
 Once you have a PR, you can do this for both C and J as part of the pre-merge CI tests
 by adding the following as a comment in the pull request:<br>
-`/azp run CI-Exhaustive` (the exhaustive tests are not run automatically on every PR).
+`run exhaustive tests` (the exhaustive tests are not run automatically on every PR).
```

Comment:
Please change to
```
Exhaustive tests should always be run for a CLDR-ICU integration PR before it is merged.
Once you have a PR, you can do this for both C and J as part of the pre-merge CI tests
by manually running a workflow (the exhaustive tests are not run automatically on every PR).
See [Continuous Integration / Exhaustive Tests](../userguide/dev/ci.md#exhaustive-tests).
```
(The GitHub UI didn't allow me to make this a "suggested change" here.)

Reply:
Thanks, I seemed to have missed that.. 
Done